app/assets/stylesheets/application/components/check-box.scss

```diff
@@ -0,0 +1,103 @@
+.check-box > .info {
+  @include display(flex);
+
+  > .left {
+    @include flex-basis(40px);
+    @include flex-shrink(0); // 横幅を40pxに固定する
+
+    width: 40px;
+  }
+
+  > .right {
+    margin-left: 10px;
+    width: 100%;
+  }
+
+  > .right > .work {
+    margin-bottom: 3px;
+
+    @if $browser == mobile {
+      font-size: 12px;
+    }
+  }
+
+  > .right > .episode,
+  > .right > .no-episode {
+    @if $browser == pc {
+      font-size: 12px;
+    }
+
+    @if $browser == mobile {
+      font-size: 10px;
+    }
+  }
+
+  > .skip {
+    @include flex-basis(60px);
+    @include flex-shrink(0);
+
+    color: $color-gray-dark-light;
+    font-size: 10px;
+    text-align: right;
+    width: 60px;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  > .skip > i {
```

Comment:
Merge rule `> .skip > i` with rule on line 35<br>Selector should have depth of applicability no greater than 3, but was 4